docs/reference/module-api/module-overview.md

```diff
@@ -840,7 +840,7 @@ module.exports = {
 
 ### `queries(self, query)`
 
-The `queries` function is part of the Apostrophe API for querying the database. It takes two arguments: the module (`self`) and the query that will be called (`query`). It must return an object. That object can have two properties:
+The `queries` function is part of the Apostrophe API for querying the database. It takes two arguments: the module (`self`) and the query that is being constructed (`query`). It must return an object. That object can have two properties:
```

Comment:
I think this first sentence is going to confuse people a lot and lead to a lot of head-bashing:

"The `queries` function is part of the Apostrophe API for querying the database."

This strongly implies you can call it, and you can't.

I think this would be better:

"The `queries` function of your module is called once when Apostrophe starts up to register custom query builders and methods."